### PR TITLE
#482 Update packages to reflect released version 0.8.0

### LINKED ISSUE #482: Update packages to reflect released version
Currently our packages are still set to 0.1.0 but they should reflect the actual released version.
We should automate this with lerna as is done eg by modelserver-node or glsp-node

### PR DESCRIPTION
**Pull request by ndoschek**
- client
  - Upgrade lerna version and lerna settings
  - Add publish:prepare script to update client versions via lerna
  - Update client README
  - Update client versions to v0.8.0
  - Update backend artifact versions in client contributions and scripts
- backend
  - Add tycho-versions plugin
  - Update backend README
  - Remove Eclipse workspace warnings
  - Update backend versions to 0.8.0-SNAPSHOT
- Update Dockerfile 
  - Update backend version
  - Reduce build steps by merging git commands into one RUN statement
  - Add script to run docker via yarn
- Update README

Resolves #482

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "private": true,
+  "name": "coffee-editor-parent",
   "description": "Eclipse EMF.cloud Coffee Editor Parent",
-  "version": "0.1.0",
+  "version": "0.0.0",
```
Comment (eneufeld) [inline package.json:5]: why set it to version 0.0.0?

**Review by eneufeld — COMMENTED** [review]
Thank you. Looks good overall. I left some comments

File: client/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "version": "0.1.0",
+  "name": "coffee-editor-client-parent",
```
[the remark this answers is not in the document]
Comment (ndoschek) [reply client/package.json:3]: I set its version to 0.0.0 as the main parent package.json. See below for the reason.

**Review by eneufeld — APPROVED** [review]
Thank you!

**Pull request merged**